docs/usage.md

```diff
@@ -22,6 +23,40 @@ The **FortiAI** solution pack offers a variety of functions through its **AI Ass
 >[!NOTE]
 > With the new voice command feature, you can perform all these tasks and more.
 
+## Generating a Connector
+
+FortiAI enables users to generate a FortiSOAR connector using natural language inputs along with API specifications. This feature is especially useful for quickly building connectors without manual coding, by simply describing the connector’s functionality and providing the relevant API details.
+
+### Creating a Connector Using FortiAI
+
+1. **User Input**  
+   In the FortiAI interface, enter a prompt that clearly defines:
+   - The purpose of the connector  
+   - The actions it needs to perform  
+   - The associated API endpoint(s)  
+
+   API details can be provided in various formats, including:
+   - `curl` commands  
+   - JSON request/response structures  
+   - File attachments in formats such as `.json`, `.yml`, `.txt`, or `.pdf`
+
+2. **Code Generation**  
+   Based on the provided input, FortiAI will automatically generate the standard connector code files:
+   - `info.json`  
+   - `connector.py`  
+   - `operations.py`  
+
+3. **Output and Packaging**  
+   The generated files, along with sample playbooks, are bundled and the connector is automatically imported into the **Build Your Own Connector (BYOC)** section within FortiSOAR.
```

Comment:
connector will be imported into create tab (user will need to refersh it) to see it.
From there, user can edit it into BYOC to edit and review the connector metadata and code